IdentityCore/src/webview/embeddedWebview/challangeHandlers/ios/MSIDCertAuthHandler.m

```diff
@@ -102,6 +102,15 @@ + (BOOL)handleChallenge:(NSURLAuthenticationChallenge *)challenge
         return NO;
     }
     
+    if (s_certAuthInProgress)
```

Comment:
I don't see us setting s_certAuthInProgress back to NO for all scenarios - e.g. it seems it is not set back to NO for ASWebAuthenticationSession. Can we make sure it is getting set back to NO?

Reply:
Rookie question for @rohitnarula7176 & @oldalton  - why is MSIDCertAuthHandler MacOS only in the latest release 1.0.18 Identity Core branch, but available on iOS on the version that is referenced by both ADAL and MSAL?